package.json

```diff
@@ -95,15 +96,16 @@
     "sinon": "^3.2.1",
     "sinon-chai": "^2.13.0",
     "sinon-express-mock": "^1.3.0",
+    "sonar-scanner": "^3.0.3",
     "supertest": "^3.0.0",
     "tslint": "^5.5.0",
     "tslint-config-standard": "^6.0.1"
   },
   "nyc": {
     "include": [
```

Comment:
why did we had test before? why it's no more used

Reply:
if you mean in  nyc area - yes it was not configured in my earlier pull request